Makefile

```diff
@@ -634,7 +634,6 @@ ovn_cleanup: ## deletes the operator, but does not cleanup the service resources
 
 .PHONY: ovn_deploy_prep
 ovn_deploy_prep: export KIND=.*
-ovn_deploy_prep: export IMAGE=unused
```

Comment:
@kajinamit so if not exported here ( just picking ovn as an example), you don't "produce" any kustomization for the service, right?

Reply:
That is correct. Previously we had to export IMAGES=usused in case we don't want to generate kustomization. This PR (actually its 2nd commit) makes this as default so you don't have to export IMAGES in any layer in case you don't need to kustomize containerImage spec

Reply:
+1 on this

Reply:
sorry I missed where `IMAGES=unused` is defined as default.
Do we need something like `IMAGE ?=unused` somewhere?

Reply:
The default is now defined here https://github.com/openstack-k8s-operators/install_yamls/pull/217/files#diff-ff72522ca932d763b97b0fadb34a28a71de16c34626f4c53810a2ffd397b1842R38

Reply:
ack thanks, so obvious :D